apps/console/src/features/applications/models/application.ts

```diff
@@ -653,6 +654,25 @@ export const emptySAMLAppConfiguration = (): SAMLApplicationConfigurationInterfa
     ssoUrl: ""
 });
 
+export type FederatedConflictWithSMSOTPArgs = {
+    /**
+     * This parameter should only pass in the configured federated
+     * authenticators under a tenant.
+     */
+    federatedAuthenticators: GenericAuthenticatorInterface[];
+    /**
+     * All the steps in the authentication sequence. Callee must pass
+     * all the authentication options without skipping any.
+     */
+    steps: AuthenticationStepInterface[];
+    subjectStepId: number;
+};
+
+export type FederatedConflictWithSMSOTPReturnValue = {
+    conflicting: boolean;
+    idpList: GenericAuthenticatorInterface[];
+}
```

Comment:
Shall we use an interface instead of type?

Reply:
Resolves via [898f5e2](https://github.com/wso2/identity-apps/pull/3782/commits/898f5e2aabd28d5fb34b94de18af0516e74ee418)